docs/evm/Build/fees.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Fees
+sidebar_label: Fees
+sidebar_position: 2
+---
+
+# Fees
+
+Transactions on FlowEVM cost FLOW token as gas. For externally owned accounts (EOAs), the interface for paying gas on FlowEVM functions the same as other EVM networks.
+
+## Technical Details
+
+Transactions on FlowEVM are executed through Cadence, which uses FLOW token for gas. Therefore, each transaction on FlowEVM has it's gas cost paid by the payer of the Cadence transaction that executes it. The account signing as the payer of the Cadence transaction therefore pays for gas of underlying FlowEVM transactions.
+
+For transactions that are submitted to a FlowEVM RPC node, the RPC node wraps the EVM transaction in a Cadence transaction and signs that transaction as the Payer. The coinbase of the EVM transaction is set by the RPC node to an address it maintains, therefore billing the cost of the transaction to the address executing it.
```

Comment:
```suggestion
Standard EVM clients such as Metamask sign and submit their transactions via the a EVM JSON-RPC network endpoint identically to other EVMs, using the FLOW token for gas.
```

I don't think we should expose the Cadence specifics, the internals of the coinbase address or "wraps the EVM transaction in a Cadence transaction and signs that transaction as the Payer" so soon. These are technical details that can come later and which apply to specific audiences beyond native EVM